rpc/backend/chain_info_test.go

```diff
@@ -295,13 +295,13 @@ func (suite *BackendTestSuite) TestGlobalMinGasPrice() {
 		expPass        bool
 	}{
 		{
-			"fail - Can't get FeeMarket params",
+			"pass - get GlobalMinGasPrice",
```

Comment:
Why this test has been changed from failing on FeeMarket to pass on GasPrice?

Reply:
we're not using the feeMarket query client for that function anymore. We're using the evm queryClient directly